File: iprep-api/Models/Response/SearchResponse.cs
namespace iprep_api.Models.Response
{
    public class SearchResponse
    {
        public int MapperId { get; set; }
        public int SubjectId { get; set; }
        public string Subject { get; set; } = null!;
        public string Question { get; set; } = null!;
        public string Answer { get; set; } = null!;
    }
}

File: iprep-frontend/src/Models/SearchResponse.ts
export class SearchResponse {
  mapperId: number;
  subjectId: number;
  subject: string;
  question: string;
  answer: string;
  constructor(
    mapperId: number,
    subjectId: number,
    subject: string,
    question: string,
    answer: string
  ) {
    this.mapperId = mapperId;
    this.subjectId = subjectId;
    this.subject = subject;
    this.question = question;
    this.answer = answer;
  }
}
